public/static/js/spawned_hosts.js

```diff
@@ -384,6 +437,12 @@ mciModule.controller('SpawnedHostsCtrl', ['$scope','$window', '$timeout', 'mciSp
         if ($scope.modalOption === 'terminateHost') {
           $scope.terminateHost();
           $('#spawn-modal').modal('hide');
+        } else if ($scope.modalOption === 'stopHost') {
+          $scope.stopHost();
+          $('#spawn-modal').modal('hide');
+        } else if ($scope.modalOption === 'startHost') {
+          $scope.startHost();
+          $('#spawn-modal').modal('hide');
```

Comment:
These and the above can probably combined into a single switch-case on the action